tests/integration/services/test_internal.py

```diff
@@ -61,3 +61,18 @@ def test_get(self):
         assert doc["machine_id"]
         assert doc["system"]
         assert type(doc["is_license_activated"]) == bool
+
+
+@pytest.mark.usefixtures("openapi_validate")
```

Comment:
Kudos for adding a test 💯 !! This fixture won't have an effect till we define the endpoint spec in `openapi.yml`. I can take care of that in a second iteration!